experimental/packages/opentelemetry-sdk-node/src/sdk.ts

```diff
@@ -193,31 +199,11 @@ export class NodeSDK {
     };
   }
 
-  /**Set configurations needed to register a LoggerProvider */
-  public configureLoggerProvider(config: LoggerProviderConfig): void {
-    // nothing is set yet, we can set config and then return
-    if (this._loggerProviderConfig == null) {
-      this._loggerProviderConfig = config;
-      return;
-    }
-
-    // make sure we do not override existing logRecordProcessor with other logRecordProcessors.
-    if (
-      this._loggerProviderConfig.logRecordProcessor != null &&
-      config.logRecordProcessor != null
-    ) {
-      throw new Error(
-        'LogRecordProcessor passed but LogRecordProcessor has already been configured.'
-      );
-    }
-
-    // set logRecordProcessor, but make sure we do not override existing logRecordProcessors with null/undefined.
-    if (config.logRecordProcessor != null) {
-      this._loggerProviderConfig.logRecordProcessor = config.logRecordProcessor;
-    }
-  }
-
-  /** Set configurations needed to register a MeterProvider */
+  /**
```

Comment:
Hi. thank you for handling this :) 
I'm not sure we should remove `configureLoggerProvider`. Can't remember we talked about this in the SIG.